rollup-bridge-contracts/forge_command_proxy.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
where used this script?

Reply:
It is for manual unittesting without nix.
If you think, it is not required, I can delete it